lib/galaxy/datatypes/sniff.py

```diff
@@ -253,75 +254,75 @@ def is_column_based( fname, sep='\t', skip=0, is_multi_byte=False ):
     return True
 
 
-def guess_ext( fname, sniff_order=None, is_multi_byte=False ):
+def guess_ext( fname, sniff_order, is_multi_byte=False ):
```

Comment:
I believe I have fixed this, we need testing of the multi-byte char stuff - unfortunately I don't have an even rudimentary understanding of the underlying issues or why those branches exist in the code.